demisto_sdk/__main__.py

```diff
@@ -330,9 +330,22 @@ def validate(config, **kwargs):
 @click.option('--cpus',
               help='Number of cpus/vcpus availble - only required when os not reflect number of cpus (CircleCI'
                    'allways show 32, but medium has 3.', hidden=True, default=os.cpu_count())
-def create_arifacts(**kwargs) -> int:
+@click.option('-idp', '--id_set_path', help='The full path of id_set.json')
+@click.option('-p', '--pack_names',
+              help=("Packs to create artifacts for. Optional values are: `all` or "
+                    "csv list of packs. "
+                    "Default is set to `all`"),
+              default="all")
```

Comment:
Don't think this is good. Think also that a partner may want to use this to create a pack that is upload-able to his local server. I would rather this be explicit.

Also please add support for the `-i` option where a directory can be specified as we have in other comamnds.